src/components/category-form/category-form.js

```diff
@@ -0,0 +1,249 @@
+    categoryImage,
+    setCategoryImage
+  } = useCategoryHandlers();
+  const languageTabs =
+    languages.length > 0
+      ? languages.map((lang) => <Tab label={lang} data-cy={lang} key={lang} />)
+      : null;
+
+  const categoryValidationSchema = Yup.object().shape({
+    enName: Yup.string()
+      .min(2, CATEGORY_VALIDATION_ERROR)
+      .required(CATEGORY_ERROR_MESSAGE),
+    uaName: Yup.string()
+      .min(2, CATEGORY_VALIDATION_ERROR)
+      .required(CATEGORY_ERROR_MESSAGE),
+      code: Yup.string()
+      .min(2, CATEGORY_VALIDATION_ERROR)
+      .required(CATEGORY_ERROR_MESSAGE)
+  });
+
+  const {
+    values,
+    handleSubmit,
+    handleChange,
+    touched,
+    errors,
+    setFieldValue
+  } = useFormik({
+    validationSchema: categoryValidationSchema,
+    initialValues: {
+      categoryImage: category.images.thumbnail || '',
+      uaName: category.name[0].value || '',
+      enName:category.name[1].value || '',
+      code:category.code || ''
+    },
+    onSubmit: (data) => {
+      const newCategory = createCategory(data);
+       if (edit) {
+        dispatch(updateCategory({ id,category: newCategory,upload }));
+        return;
+      }
+      dispatch(addCategory({category: newCategory,upload }));
+     
+    }
+  });
+
+  const handleImageLoad = (e) => {
+    if (e.target.files && e.target.files[0]) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        setFieldValue('categoryImage', event.target.result);
+        setCategoryImage(event.target.result);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+      setUpload(e.target.files[0]);
+    }
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <Grid item xs={12}>
+          <Paper className={styles.categoryItemUpdate}>
+            <span className={styles.imageUpload}>
+              {config.labels.avatarText}
+            </span>
+            <ImageUploadContainer
+              handler={handleImageLoad}
+              srcForAvatar={
+                categoryImage ||
+                `${config.imagePrefix}${values.categoryImage}`
+              }
+                fileName={upload.name ||category.images.thumbnail}
+            />
+            <TextField
+              data-cy='code'
+              name='code'
+              className={styles.textField}
+              variant='outlined'
+              label={'Код категорії'}
+              value={values.code}
+              onChange={handleChange}
+              error={touched.code && !!errors.code}
+            />
+            {touched.code && errors.code && (
+              <div data-cy='code-error' className={styles.inputError}>
+                {errors.code}
+              </div>
+            )}
+          </Paper>
+        </Grid>
+        <AppBar position='static'>
+          <Tabs
+            className={styles.tabs}
+            value={tabsValue}
+            onChange={handleTabsChange}
+            aria-label='simple tabs example'
```

Comment:
Why you set this in `aria-label`?

Reply:
It was probably left from examlpe from material.ua site/
I all ready removed it